lavalink/client.py

```diff
@@ -69,9 +69,10 @@ def add_event_hook(self, hook, event: Event = None):
             dispatched.
         """
         event_hooks = self._event_hooks.get(event)
+
         if hook not in event_hooks:
             if event_hooks is None:
-                self._event_hooks[event] = [hook]
+                self._event_hooks["Generic"] = [hook]
```

Comment:
FWIW this line is also bugged in the sense that if an Event type is specified but the array is empty, then the hook will be registered for all events rather than the specific type.